build.sbt

```diff
@@ -1,47 +1,152 @@
+  .settings(publishingSettings: _*)
+  .settings(defaultSettings(): _*)
+  .settings(scalariformSettings: _*)
+  .settings(wartRemoverSettings)
+  // disable some wart remover checks in tests - (Any, Null, PublicInference) seems to struggle with
+  // scalamock, (Equals) seems to struggle with stub generator AutoGen and (NonUnitStatements) is
+  // imcompatible with a lot of WordSpec
+  .settings(wartremoverErrors in (Test, compile) --= Seq(Wart.Any, Wart.Equals, Wart.Null, Wart.NonUnitStatements, Wart.PublicInference))
+  .settings(wartremoverExcluded ++=
+    routes.in(Compile).value ++
+      (baseDirectory.value ** "*.sc").get ++
+      (baseDirectory.value ** "HealthCheck.scala").get ++
+      (baseDirectory.value ** "HealthCheckRunner.scala").get ++
+      (baseDirectory.value ** "Lock.scala").get ++
+      Seq(sourceManaged.value / "main" / "sbt-buildinfo" / "BuildInfo.scala")
+  )
+  .settings(
+    libraryDependencies ++= Seq(
+        ws,
+        "uk.gov.hmrc" %% "auth-client" % "2.5.0",
+        "uk.gov.hmrc" %% "microservice-bootstrap" % "6.15.0",
+        "uk.gov.hmrc" %% "play-config" % "4.3.0",
+        "uk.gov.hmrc" %% "domain" % "5.1.0",
+        "org.typelevel" %% "cats" % "0.9.0",
+        "com.github.kxbmap" %% "configs" % "0.4.4",
+        "uk.gov.hmrc" %% "play-reactivemongo" % "6.1.0",
+        "com.eclipsesource" %% "play-json-schema-validator" % "0.8.9",
+        "uk.gov.hmrc" %% "mongo-lock" % "5.0.0",
+
+        "uk.gov.hmrc" %% "hmrctest" % "3.0.0" % test,
+        "org.scalatest" %% "scalatest" % "3.0.4" % test,
+        "org.pegdown" % "pegdown" % "1.6.0" % test,
+        "com.typesafe.play" %% "play-test" % PlayVersion.current % test,
+        "com.github.tomakehurst" % "wiremock" % "2.5.1" % test,
+        "org.scalamock" %% "scalamock-scalatest-support" % "3.5.0" % test,
+        "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.0" % test,
+        "com.miguno.akka" % "akka-mock-scheduler_2.11" % "0.5.1" % test,
+        "com.typesafe.akka" %% "akka-testkit" % "2.3.11" % test,
+        "uk.gov.hmrc" %% "stub-data-generator" % "0.4.0" % test
+    )
+  )
+  .settings(version := "0.0.1",
+    scalaVersion := "2.11.11",
+    credentials += Credentials(Path.userHome / ".sbt" / ".credentials"),
+    scalacOptions ++= Seq("-unchecked", "-deprecation")
+  )
+  .settings(
+    retrieveManaged := true,
+    evictionWarningOptions in update := EvictionWarningOptions.default.withWarnScalaVersionEviction(false),
+    routesGenerator := StaticRoutesGenerator
+  )
+  .settings(resolvers ++= Seq(
+    Resolver.bintrayRepo("hmrc", "releases"),
+    Resolver.jcenterRepo,
+    Resolver.bintrayRepo("hmrclt", "maven"),
+    "hmrc-snapshots" at hmrcRepoHost + "/content/repositories/hmrc-snapshots",
+    "hmrc-releases" at hmrcRepoHost + "/content/repositories/hmrc-releases",
+    "typesafe-releases" at hmrcRepoHost + "/content/repositories/typesafe-releases"
+  ))
+
+  def oneForkedJvmPerTest(tests: Seq[TestDefinition]) =
```

Comment:
I do not see the usages of `oneForkedJvmPerTest` in this file , either you missed the `IntegrationTest` Config or something